ludwig/models/modules/sequence_decoders.py

```diff
@@ -149,104 +149,56 @@ class SequenceTaggerDecoder(Layer):
     def __init__(
             self,
             initializer=None,
-            regularize=True,
+            use_bias=True,
+            kernel_initializer='glorot_uniform',
+            bias_initializer='zeros',
+            kernel_regularizer=None,
+            bias_regularizer=None,
+            activity_regularizer=None,
             attention=False,
             num_classes=0,
-            loss=None,
+            is_timeseries=None,
             **kwargs
     ):
         super(SequenceTaggerDecoder, self).__init__()
         self.initializer = initializer
-        self.regularize = regularize
         self.attention = attention
 
-        # todo tf2 add other required parameters for Dense layer
-        self.decoder_layer = Dense(num_classes)
+        if is_timeseries:
+            units = 1
```

Comment:
would simplify to `num_classes = 1` and remove the else

Reply:
done